.ruby-version

```diff
@@ -0,0 +1 @@
+3.0.2
```

Comment:
We don't need this file because we specify Ruby version to `ruby/setup-ruby` explicitly.

Reply:
this will help developers locally

Reply:
OK.
So, please remove the `ruby-version` argument from `ruby/setup-ruby`. If we don't specify the `ruby-version` argument, Ruby version is read from `.ruby-version`.